service/src/framework/github.ts

```diff
@@ -293,3 +293,32 @@ export const createOrUpdatePRComment = async ({
     return { result: 'failure', message: (err as Error).message || 'Failed to create PR comment' };
   }
 };
+
+interface IsApproveCommentFoundParams {
+  owner: string;
+  repo: string;
+  prNumber?: string;
+  installationOctokit: Octokit;
+  log: FastifyLoggerInstance;
+}
+
+export async function isApproveCommentFound({
```

Comment:
We would also want to only be looking at comments that were created after the last commit was pushed to the PR (or since the most recent Bundlemon run for the PR).